pytest_html/plugin.py

```diff
@@ -157,24 +158,24 @@ def create_asset(self, content, extra_index,
         def append_extra_html(self, extra, extra_index, test_index):
             href = None
             if extra.get('format') == extras.FORMAT_IMAGE:
-                if self.self_contained:
+                content = extra.get('content')
+                if re.match('^file|^http', content):
```

Comment:
Not sure what you mean... What won't cover absolute or relative paths? What other types are there? While testing this I used an image with an absolute path.

Do you want to cover relative paths as well?